stl_to_obj/textures.py

```diff
@@ -0,0 +1,48 @@
+import json
+
+# filenames of all the PBR textures included in game in the textures folder, stored as a dict for each file
+TEXTURE_FOLDER = "textures"
+
+NORMAL_OPTIONS = "normal_enabled = true\nnormal_scale = -1.0\n"
+DEPTH_OPTIONS = "depth_enabled = true\ndepth_scale = -0.01\ndepth_deep_parallax = false\ndepth_flip_tangent = false\ndepth_flip_binormal = false\n"
+
+class Texture:
+    def __init__(self, folder, jpg_dict, uv1_scale = ""):
+        self.folder = folder
+        self.jpg_dict = jpg_dict
+        # uv shouldn't be in the dict because it doesn't need an ext_resource like the .jpgs
+        self.uv1_scale = uv1_scale 
+
+# dict of all the textures we already have
+texture_dict = dict()
+other_textures = dict()
+
+# add textures from a .json file to dict 
+# allows contributors to use their own textures (that we don't have)
+def load_textures(filename):
+    data = ""
```

Comment:
I don't think you need this since you initialize `data` with file data.

Reply:
You're right, this entire time I was under the impression that data would go out of scope after the with block if I didn't initialize it before the with block.